finetune/base_models/bert/model.py

```diff
@@ -484,14 +485,14 @@ class TableRoBERTa(_BaseBert):
     # Due to use of ragged tensors we cannot support XLA for this model.
     # Wrapping the inner call method in a tf.function with jit_compile=False causes missing gradient errors that I don't understand.
     # Longeer term we should try to refactor this to use a more XLA friendly approach.
-    supports_xla = False
+    supports_xla = True
     settings = {
         **BERT_BASE_PARAMS,
         # Just incase all cells fall into the same buckets this -8 allows us to pack the batches much tighter once we add EOS and BOS
         "max_length": 2048 - 8,
-        "table_batching": True,
+        "table_batching": False,
         "chunk_tables": True,
-        "batch_size": 2,  # When table batching is true this becomes a nominal batch size. Very long docs have batch size = 1
+        "batch_size": 1,  # When table batching is true this becomes a nominal batch size. Very long docs have batch size = 1
```

Comment:
Batch size needs to be dropped b/c when we have to pad rather than using ragged tensors this leads to a lot of padding?